src/node/biquad_filter.rs

```diff
@@ -518,18 +518,28 @@ impl BiquadFilterNode {
         //
         // with z1 = 1/z and z = exp(j*pi*frequency). Hence z1 = exp(-j*pi*frequency)
         for (i, freq) in frequency_hz.iter().enumerate() {
```

Comment:
```suggestion
        for (i, &freq) in frequency_hz.iter().enumerate() {
```

Then you can leave out `let freq = *freq;`

Reply:
Done!